brion/morphology.h

```diff
@@ -81,6 +86,14 @@ class Morphology : public boost::noncopyable
      */
     BRION_API Vector2isPtr readApicals() const;
 
+    /**
+     * @return perimeters of the cross sections for each point of the morphology
+     *         in micrometers.
+     * @throw std::runtime_error if empty for FAMILY_GLIA
```

Comment:
Didn't we agree that the existence of the dataset should be checked at construction?
But this is also fine, because I don't think we are checking the consistency of any file in the constructor.